srt-protocol/src/settings/connection.rs

```diff
@@ -59,6 +61,7 @@ impl From<options::SocketOptions> for ConnInitSettings {
             bandwidth: options.sender.bandwidth,
             statistics_interval: options.session.statistics_interval,
             recv_buffer_size: options.receiver.buffer_size / options.session.max_segment_size,
+            send_buffer_size: options.sender.buffer_size / options.session.max_segment_size,
```

Comment:
Is buffer_size supposed to be how many data payload bytes? If so it should be `buffer_size / (mss - headers)`

Reply:
Good point, I checked the options documentation and the receive buffer size limit should also be calculated this way too.


> SRTO_RCVBUF
> 
> Receive Buffer Size, in bytes. Note, however, that the internal setting of this value is in the number of buffers, each one of size equal to SRT payload size, which is the value of SRTO_MSS decreased by UDP and SRT header sizes (28 and 16). The value set here will be effectively aligned to the multiple of payload size.